scopes/component/component/aspect-list.ts

```diff
@@ -81,6 +81,12 @@ export class AspectList {
     return serializedEntries;
   }
 
+  filter(ids?: string[]) {
```

Comment:
If you call it filter, you should get a callback function to be consistent with the filter.
If you want to filter by ids, call it filterByIds.